LibraryManagementSystem/Forms/DeleteTab.cs

```diff
@@ -80,8 +80,9 @@ private void btnDelete_Click(object sender, EventArgs e)
                     // checks if user chooses the "yes" option in the message box
                     if (result == DialogResult.Yes)
                     {
-                        cmd0 = new SqlCommand("DELETE FROM booksData WHERE BookID = '" + bookID + "'", con);
+                        cmd0 = new SqlCommand("DELETE FROM book WHERE book_id = '" + bookID + "'", con);
                         cmd0.ExecuteNonQuery();
+                        cmd1 = new SqlCommand("DELETE FROM borrow_data WHERE book_id = '" + bookID + "'", con);
```

Comment:
Shouldn't inserting values into borrow_data only happen on ReturnTab?

Reply:
Nope. There's a one-to-one relationship between book.book_id and borrow_data.book_id. I haven't constrained it (I'll work that out ig, but it'll be super redundant) and so you kinda need an identifier from one table to another. cmd1 creates said identifier.